### PR TITLE
feat: add multi_scm option for build proto

### PR DESCRIPTION
**Pull request by chingor13**
This will allow us to trigger Kokoro jobs that use `multi_scm`.

To run with a multi_scm config: `python3 -m autorelease trigger-single --pull=<pull-request-url> --multi-scm-name=<scm-name>`. It will create a Kokoro build request that uses the repo's base name as the `scm_name`.

### CODE REVIEW

File: autorelease/kokoro.py [hunk 1]
```diff
@@ -42,13 +43,18 @@ def _send_pubsub_message(
...
+def _make_build_request(
+    job_name: str, sha: str, env_vars: dict = None, multi_scm_name: Optional[str] = None
+) -> str:
     request = kokoro_api_pb2.BuildRequest(
         full_job_name=job_name,
-        scm_revision=kokoro_api_pb2.ScmRevision(
-            github_scm_revision=kokoro_api_pb2.GithubScmRevision(commit_sha=sha)
-        ),
     )
+    if multi_scm_name:
```
Comment (SurferJeffAtGoogle) [inline autorelease/kokoro.py:52]: Please add a comment explaining why the code does what it does.

File: autorelease/kokoro.py [hunk 2]
```diff
@@ -42,13 +43,18 @@ def _send_pubsub_message(
     return resp
 
 
-def _make_build_request(job_name: str, sha: str, env_vars: dict = None) -> str:
+def _make_build_request(
+    job_name: str, sha: str, env_vars: dict = None, multi_scm_name: Optional[str] = None
```
Comment (SurferJeffAtGoogle) [inline autorelease/kokoro.py:47]: Would would it mean if a caller passed an empty string for multi_scm_name.  Would it mean the same as `None`?

To reduce this ambiguity, the declaration could be changed to:
`, multi_scm_name: str = ""`

**Review by SurferJeffAtGoogle — CHANGES_REQUESTED** [review]

File: autorelease/kokoro.py (hunk 1 above)
Comment (chingor13) [reply autorelease/kokoro.py:54]: Done.

File: autorelease/kokoro.py (hunk 2 above)
Comment (chingor13) [reply autorelease/kokoro.py]: Done

**Review by SurferJeffAtGoogle — APPROVED** [review]

**Pull request merged**